grid-plot/gridPlot.js

```diff
@@ -0,0 +1,123 @@
+import * as d3 from 'd3';
+import gChartcolour from 'g-chartcolour';
+
+export function draw() {
+    let yScale = d3.scaleBand();
+    let xScale = d3.scaleBand();
+    let groupNames = [];
+    let yAxisAlign = 'right';
+    let rem = 16;
+    const colourScale = d3.scaleOrdinal()
+        .domain(groupNames);
+
+    function chart(parent) {
+        xScale.paddingInner(0);
```

Comment:
What do you think about having this an option/variable in the index file?

Reply:
Personally I don't think it's necessary. It would make it harder to add a separator in Illustrator in between the categories. Thanks for going thru it!